README.md

```diff
@@ -11,12 +13,11 @@
 
 <!-- See the [documentation](https://monad-bayes.netlify.app/) for a quick-start user guide and a reference overview of how it all works. -->
 
-Created by [Adam Scibior][adam-web] ([@adscib][adam-github]), documentation, website and various extras by [Reuben][reuben-web], maintained by [Tweag I/O][tweagio].
+Created by [Adam Scibior][adam-web] ([@adscib][adam-github]), documentation, website and newer features by [Reuben][reuben-web], maintained by [Tweag I/O][tweagio].
```

Comment:
```suggestion
Created by [Adam Scibior][adam-web] ([@adscib][adam-github]), documentation, website and newer features by [Reuben][reuben-web], maintained by [Tweag][tweagio].
```

Reply:
Here is the current draft of the website: https://monad-bayes-site.netlify.app/_site/about.html I am currently using my own netlify to host it. I would definitely prefer to switch this to the Tweag Netlify, but I'll need access to do that. I'm happy with any url.

Reply:
@garbas please advise.

Reply:
Sorry for late reply.

We can of course move this to Tweag's Netlify, but I would encourage you to look at Cloudflare Pages (this is what we are using for tweag.io) since it provides much better services.

Reply:
How do we provide hosting for https://nickel-lang.org/, for instance? This is a similar situation.

Reply:
I've finished my fellowship at Tweag, so while I'm happy to hand over the page to Tweag's cloudflare or netlify, I don't have much time to work out the logistics. My suggestion is we put this aside for the current PR.